web/templates/keyboard_shortcuts.hbs

```diff
@@ -258,6 +258,10 @@
                     <td class="definition">{{t 'Toggle topic mute' }}</td>
                     <td><span class="hotkey"><kbd>Shift</kbd> + <kbd>M</kbd></span></td>
                 </tr>
+                <tr>
```

Comment:
I wonder if this is the right location for documenting this keyboard shortcut? To my mind, the history belongs up by the edit/view source and move icons. I'm curious to see what others thing--there doesn't seem to be a whole lot of rhyme or reason as to why things are ordered as they are here, but putting this at the very bottom doesn't seem quite right.

Reply:
Yeah this is wrong. I'll do an initial tweak so that I feel comfortable merging the feature change, and also opened https://github.com/zulip/zulip/issues/26573 for related follow-up work here; @drrosa can take care of making this nice as a follow-up as he maintains our /help/ documentation.